www/front_src/src/Resources/Graph/Performance/Graph/Axes/Y.tsx

```diff
@@ -29,7 +29,7 @@ const UnitLabel = ({ x, unit }: UnitLabelProps): JSX.Element => {
   return (
     <text
       x={x}
-      y={0}
+      y={-8}
```

Comment:
It's to adjust this value forced ?

Reply:
So that it's a bit elevated compared to the Axis